homeassistant/components/twinkly/light.py

```diff
@@ -35,7 +45,10 @@ async def async_setup_entry(
 ) -> None:
     """Setups an entity from a config entry (UI config flow)."""
 
-    entity = TwinklyLight(config_entry, hass)
+    client = hass.data[DOMAIN][config_entry.data[CONF_ENTRY_ID]][DATA_CLIENT]
```

Comment:
Would be good to rename `CONF_ENTRY_ID` to something that is less mistakable for config entry entry_id.